### PR TITLE
Move policies/tenant_cluster.json to policies/aws-operator.json

### PR DESCRIPTION
**Pull request by kopiczko**
Towards https://github.com/giantswarm/giantswarm/issues/12945

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - `RouteTables` flag will be deprecated.
 
+### Documentation
+
+- Move policies/tenant_cluster.json to policies/aws-operator.json as they are
+  supposed to be used for both Control Plane and Tenant Clusters AWS accounts.
```
Comment (kopiczko) [inline CHANGELOG.md:33]: I'm happy to revert this if doesn't make sense. See https://gigantic.slack.com/archives/C0251MXL5/p1599039594007700

Comment (xh3b4sd) [reply CHANGELOG.md:33]: Not blocking. Just for my understanding, does this align with the semver spec? Or is this some arbitrary block we put here?

Comment (kopiczko) [reply CHANGELOG.md:33]: This is my invention after I saw this https://github.com/kubernetes-sigs/cluster-api/releases
Which is not the best example of semver compliance.

Comment (xh3b4sd) [reply CHANGELOG.md:33]: I would not add these arbitrary sections then. It is just getting more complicated. You look at the changelog for contract modifications. You look at the commits for everything else. That is as straight forward as it gets.

Comment (kopiczko) [reply CHANGELOG.md]: Removed.

**Review by xh3b4sd — APPROVED** [review]

**Pull request merged**